### ISSUE TITLE
bootstrap-table.min.js

### ISSUE THREAD

**Issue by abriz** [wenzhixin/bootstrap-table#606]
For some reason, my selectAll Checkbox column (data-checkbox) isn't working, no checkbox appear, when using the minified js file.  When using the non minified file (boostrap-table.js) all works fine.  I am using version 1.5.0.  Anyone else having this same problem?

**Comment by djhvscf** [comment]
oh! let me check this problem.. @abriz

**Comment by djhvscf** [comment]
For me that feature works in 1.5.0 min version.. share your code please! @abriz

**Comment by abriz** [comment]
I appreciate you checking.  Let me try again with a bare bones new project and see if I can repro outside of my main project.

**Comment by djhvscf** [comment]
ok, if not, let me know @abriz

**Comment by abriz** [comment]
Will do!
Thanks again.

Date: Tue, 10 Mar 2015 12:40:32 -0700

**Comment by wenzhixin** [comment]
Update to the latest version.
